SelectRequestInput.module.scss

```diff
@@ -0,0 +1,7 @@
+.api-request {
+    display: flex;
+    flex-wrap: nowrap;
+    border-radius: 4px;
+    margin-bottom: 24px;
+    border: 0.5px solid #252525b8;
```

Comment:
Perhaps we can use a scss color variable for this? If it doesn't exist, make a new one for it and maybe call it something like `api-border`